js/stores/eventStore.js

```diff
@@ -98,8 +98,10 @@ const doAction = (action) => {
       eventState = eventState.set('page_load', eventState.get('page_load').push(pageLoadEvent))
       break
     case WindowConstants.WINDOW_SET_FOCUSED_FRAME:
-      addPageView(action.frameProps.get('location'))
-      lastActiveTabId = action.frameProps.get('tabId')
+      if (action.frameProps) {
```

Comment:
It's the same fix, just in a different place, but I can make it if you want

> On Aug 29, 2016, at 1:52 PM, Marshall T. Rose <notifications@github.com> wrote:
> 
> In js/stores/eventStore.js:
> 
> > @@ -98,8 +98,10 @@ const doAction = (action) => {
> >        eventState = eventState.set('page_load', eventState.get('page_load').push(pageLoadEvent))
> >        break
> >      case WindowConstants.WINDOW_SET_FOCUSED_FRAME:
> > -      addPageView(action.frameProps.get('location'))
> > -      lastActiveTabId = action.frameProps.get('tabId')
> > +      if (action.frameProps) {
> ok, who should we re-assign it to in order to implement a more general solution?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>